src/integrations/__tests__/unixIntegrationManager.spec.ts

```diff
@@ -47,287 +43,260 @@ afterEach(async() => {
   }
 });
 
-testUnix('Ensure symlinks and dirs are created properly', async() => {
-  const integrationManager = new UnixIntegrationManager(
-    resourcesDir, integrationDir, dockerCliPluginDir);
+describeUnix('UnixIntegrationManager', () => {
+  let integrationDir: string;
+  let dockerCliPluginDir: string;
+  let integrationManager: UnixIntegrationManager;
 
-  await integrationManager.enforce();
-  expect(fs.promises.readdir(integrationDir)).resolves.not.toThrow();
-  for (const name of await fs.promises.readdir(resourcesDir)) {
-    const integrationPath = path.join(integrationDir, name);
+  beforeEach(() => {
+    integrationDir = path.join(testDir, INTEGRATION_DIR_NAME);
+    dockerCliPluginDir = path.join(testDir, 'dockerCliPluginDir');
+    integrationManager = new UnixIntegrationManager(
+      resourcesDir, integrationDir, dockerCliPluginDir);
+  });
 
-    expect(fs.promises.readlink(integrationPath, 'utf8')).resolves.not.toThrow();
-  }
-  for (const name of await integrationManager.getDockerCliPluginNames()) {
-    const pluginPath = path.join(dockerCliPluginDir, name);
+  test('.enforce() should create dirs and symlinks properly', async() => {
+    await integrationManager.enforce();
+    expect(fs.promises.readdir(integrationDir)).resolves.not.toThrow();
```

Comment:
Missing `await`, and I prefer this test:
```
await expect(fs.promises.readlink(integrationPath, 'utf8'))
        .resolves.toMatch(new RegExp(`^resources/.+/${ name }$`));
```

Reply:
Deleted this line. See https://github.com/rancher-sandbox/rancher-desktop/pull/3183/files/242bcdc669e33c75021d6956bc8e871860d75f6c#r999707637